fmriprep/interfaces/freesurfer.py

```diff
@@ -190,36 +192,71 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class TruncateLTA(object):
+    # Use a tuple in case some object produces multiple transforms
+    lta_outputs = ('out_lta_file',)
+
+    def _post_run_hook(self, runtime):
+
+        outputs = self.aggregate_outputs(runtime)
+        print(outputs)
+        print(type(outputs))
+        # I will modify this as soon as I understand what is the right syntax to use it
+        outputs = self._list_outputs()
```

Comment:
I still do not understand how to change this one
if I use 
outputs = self.aggregate_outputs(runtime)
then I get an error with the line
    lta_file = outputs[lta_name]
TypeError: 'MRICoregOutputSpecRPT' object is not subscriptable

lta_name can be the string 'out_file' or 'out_lta_file'

if I understand from other example we should call
outputs.out_file or outputs.out_lta_file

but how to use the lta_name ?

Reply:
Okay. This seems to be a bug, because the output of `Interface.aggregate_outputs(...)` should be a `dict`-like object, not an `OutputSpec`. I'll try to track that down.

In the meantime, using `_list_outputs()` should be fine. It's a little dissatisfying to me for architectural reasons (`aggregate_outputs` can take the `runtime` object and thus have access to the working directory if the directory has changed), but I think in practice it will be fine.

Sorry for dragging this out.